strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryPath.java

```diff
@@ -152,9 +154,9 @@ public URI toUri()
         throw new UnsupportedOperationException();
     }
 
-    public Path toAbsolutePath()
+    public RepositoryPath toAbsolutePath()
     {
-        throw new UnsupportedOperationException();
+        return createByTemplate(getTarget().toAbsolutePath());
```

Comment:
Now I think it's better to be named `wrap()` instead of `createByTemplate` or `createFromTemplate`, and `getTarget()` will be `unwrap()` in this case.

this method wraps Storage File System `Path` implementation with `RepositoryPath` which is`LayoutProvider` logic.